### ISSUE TITLE
Mask loss implementation difference

### ISSUE THREAD

**Issue by sagniklp** [lioryariv/idr#11]
Hi, thank you for this repository.
I have a question about Mask loss (Eq. 7). According to the paper:
![image](https://user-images.githubusercontent.com/10586359/106827879-3dcd8880-6657-11eb-97a3-2d7bc4fd0a40.png)

But in the implementation, sigmoid is not used:
![image](https://user-images.githubusercontent.com/10586359/106827818-1e366000-6657-11eb-9de8-b38ba4383514.png)

Could you please explain why?

**Comment by lioryariv** [comment]
Hi, 
The function 'binary_cross_entropy_with_logits' combines a Sigmoid layer followed by a BCE loss (https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html).

So it is the same implementation :)

**Comment by sagniklp** [comment]
Oh sorry, my bad!
